example/winrt_geolocation.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// File Open Picker from Dart
```

Comment:
nit: This comment needs to be updated :).

Also, consider adding this example to the `README.md` file in this folder.